templates/generation/generate_release_please_config.sh

```diff
@@ -51,39 +51,8 @@ for module in $(find . -mindepth 2 -maxdepth 2 -name pom.xml |sort | xargs dirna
   #adding the line to manifest config file
   echo "${rp_manifest_line}" >> .release-please-manifest.json
 
-  #for multi-module libraries, sync the versions to released(non-snapshot):current(snapshot) type
-  cat ${version_file} | while read line
-      do
-         if [[ ${line} =~ [0-9] ]] && ! [[ ${line} == *"SNAPSHOT"* ]] ; then
-           artifact_name=$(echo "${line}" | awk -F':' '{print $1}')
-           old_version=$(echo "${line}" | awk -F':' '{print $3}')
-           new_version=$(echo ${old_version} |  awk -F'.' '{print $1"."$2"."$3+1}' |  sed s/[.]$//)
-           new_version="${new_version}-SNAPSHOT"
-           sed -i "s|${artifact_name}:${old_version}:${old_version}|${artifact_name}:${old_version}:${new_version}|" ${version_file}
-           artifact_directory="${module}/${artifact_name}"
-           mvn -B -ntp -f ${artifact_directory} -U versions:set -DnewVersion=${new_version}
-           mvn versions:commit
-         fi
-      done
-
-  snapshot_version=$(echo ${module_released_version} |  awk -F'.' '{print $1"."$2"."$3+1}' |  sed s/[.]$//)
-
-  #bumping to snapshot in module bom directory, if it exists
-  bom_directory="${module-name}/${artifactName_config}-bom"
-  if [ -d "${bom_directory}" ]; then
-    mvn -B -ntp -f ${bom_directory} -U versions:set -DnewVersion="${snapshot_version}-SNAPSHOT"
-    mvn versions:commit
-  fi
-
   #bumping to snapshot in module root pom
-  mvn -B -ntp -f ${module} -U versions:set -DnewVersion="${snapshot_version}-SNAPSHOT"
-  mvn versions:commit
-
-  #specific case
-  if [[ ${module_name} == "java-notification" ]]; then
-    mvn -B -ntp -f ${module} -U versions:set -DnewVersion="${snapshot_version}-beta-SNAPSHOT"
-    mvn versions:commit
-  fi
+  mvn -B -ntp -f ${module} -U versions:set -DnewVersion="${module_snapshot_version}" -DprocessAllModules -DgenerateBackupPoms=false -DprocessFromLocalAggregationRoot=false
```

Comment:
All sub-modules should have the same version. So, I don't think you need to go through each library sub-module one by one.

Reply:
that's the thing, they might be not. For example, in java-vision, we've got different versions for submodules -> https://github.com/googleapis/java-vision/blob/main/versions.txt

Reply:
I didn't know that was possible. I'll bring your per-module code back.